uber/models.py

```diff
@@ -1948,67 +1981,37 @@ def track(cls, action, instance):
                 action = c.AUTO_BADGE_SHIFT
             elif not data:
                 return
-        elif instance == 'Budget':  # Vaguely horrifying special-casing where we make up fake data so we can insert this entry into the tracking DB
-            data = 'Budget Page'
-            who = AdminAccount.admin_name() or (current_thread().name if current_thread().daemon else 'non-admin')
-            with Session() as session:
-                session.add(Tracking(
-                    model='Budget',
-                    fk_id=str(uuid4()),
-                    which='Budget',
-                    who=who,
-                    links='',
-                    action=action,
-                    data=data
-                ))
-            return
         else:
             data = 'id={}'.format(instance.id)
         links = ', '.join(
             '{}({})'.format(list(column.foreign_keys)[0].column.table.name, getattr(instance, name))
             for name, column in instance.__table__.columns.items()
             if column.foreign_keys and getattr(instance, name)
         )
-        who = AdminAccount.admin_name() or (current_thread().name if current_thread().daemon else 'non-admin')
+
+        if sys.argv == ['']:
```

Comment:
This equals something else 'when the server is running'. As it turns out, though, AdminAccount.admin_name() was always truthy during my testing, so if I tried to use either that or this, it would always track it as the admin I was logged in as. Even... even if I logged out. Yeah, I don't get it either.

For the record, I kept my server running when I tested this via the Python REPL, so it doesn't seem to depend on us actually shutting down the server.